glu will fail pkgconf test pipeline

If the pkgconf test pipeline is added to glu it will fail in the following way: 

```2024/11/12 23:11:01 INFO running step "test/pkgconf"
2024/11/12 23:11:02 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:11:02 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:11:02 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:11:02 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:11:02 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN + basename /home/build/melange-out/glu uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN + dev_pkg=glu uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN + apk info -L glu uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN + grep -q ^Name: usr/lib/pkgconfig/glu.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN + basename usr/lib/pkgconfig/glu.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN + lib_name=glu uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN + echo usr/lib/pkgconfig/glu.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN + grep -q '^usr/lib/pkgconfig/glu.pc$\|^usr/share/pkgconfig/glu.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN + pkgconf --exists glu uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN + pkgconf --print-errors --exists glu uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN Package osmesa was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN Perhaps you should add the directory containing `osmesa.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 WARN Package 'osmesa', required by 'glu', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:02 INFO pod 139a516bc23e44afaa07624572166627f856dcb0d653388b85158aa4f1f213b7 terminated
2024/11/12 23:11:02 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:11:02 INFO   workspace dir: /home/user/tmp/melange-workspace-1438939021
2024/11/12 23:11:02 INFO   guest dir: /home/user/tmp/melange-guest-3101997161
2024/11/12 23:11:02 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/glu] Error 1```

--- a/glu.yaml
+++ b/glu.yaml
@@ -1,7 +1,7 @@
 package:
   name: glu
   version: 9.0.3
-  epoch: 0
+  epoch: 1
   description: "Mesa OpenGL Utility library"
   copyright:
     - license: SGI-B-1.1
@@ -36,6 +36,10 @@ pipeline:
   - uses: meson/install
 
   - uses: strip
+
+test:
+  pipeline:
+    - uses: test/pkgconf
 
 update:
   enabled: true

